e2e/tests/image.test.js

```diff
@@ -138,3 +147,129 @@ Scenario('Image with perRegion tags', async function({ I, AtImageView, AtSidebar
   assert.strictEqual(result.length, 1);
   assert.deepStrictEqual(result[0].value.rectanglelabels, ['Moonwalker']);
 });
+
+Scenario('Can\'t create rectangles outside of canvas', async ({ I, AtLabels, AtSidebar, AtImageView, LabelStudio }) => {
```

Comment:
We may want to use `DataTable` to save ourselve from the regression without relative coordinates.
Something like:
```js
const ffs = new DataTable(['FF_3793']);

ffs.add([true]);
ffs.add([false]);

Data(ffs).Scenario('Can\'t create rectangles outside of canvas', async ({ I, AtLabels, AtSidebar, AtImageView, LabelStudio, current }) => {
  const { FF_3793 } = current;

  LabelStudio.setFeatureFlags({
    fflag_fix_front_dev_3793_relative_coords_short: FF_3793,
  });
```
and reuse it for the second test. As I see it passes the tests without any other changes if there is everithing ok.